docs/tutorials/03_shaders/viz_pbr_spheres.py

```diff
@@ -0,0 +1,136 @@
+"""
+===============================================================================
+Physically-Based Rendering (PBR) on spheres
+===============================================================================
+
+PBR engines aim to simulate properties of light when it interacts with objects
+in the scene in a physically plausible way. The interaction of light with an
+object depends on the material the object is made of. In computer graphics,
+materials are usually divided in 2 main categories based on their conductive
+properties: dielectrics and metals.
+
+This tutorial, illustrates how to model some material properties in FURY by
+using the PBR material.
+
+Let's start by importing the necessary modules:
+"""
+
+from fury import actor, material, window
+from fury.utils import (normals_from_actor, tangents_to_actor,
+                        tangents_from_direction_of_anisotropy)
+import numpy as np
+
+"""
+Now set up a new scene.
+"""
+
+scene = window.Scene()
+scene.background((.9, .9, .9))
+
+"""
+Let's define the parameters we are going to showcase in this tutorial.
+These subset of parameters have their values constrained in the 0 to 1 range.
+"""
+
+material_params = [
+    [[1, 1, 0], {'metallic': 0, 'roughness': 0}],
+    [(0, 0, 1), {'roughness': 0}],
+    [(1, 0, 1), {'anisotropy': 0, 'metallic': .25, 'roughness': .5}],
+    [(1, 0, 1), {'anisotropy_rotation': 0, 'anisotropy': 1, 'metallic': .25,
+                 'roughness': .5}],
+    [(0, 1, 1), {'coat_strength': 0, 'roughness': 0}],
+    [(0, 1, 1), {'coat_roughness': 0, 'coat_strength': 1, 'roughness': 0}]
+]
+
+"""
+Now we can start to add our actors to the scene and see how different values of 
+the parameters produce interesting effects.
+"""
+
+for i in range(6):
```

Comment:
what is this `6`? I recommend you to use `enumerate` with `material_params`

Reply:
Changed.